### ISSUE TITLE
New license request: Government of Japan Standard Terms of Use (Version 2.0)

### ISSUE THREAD

**Issue by sorairolake** [spdx/license-list-XML#1925]
### Introduction

This is applied to contents created by the Government Agencies of Japan.

### License Name

Government of Japan Standard Terms of Use (Version 2.0)

### Suggested short identifier

GJSTU-2.0

### URL to license text

[English version](https://www.digital.go.jp/assets/contents/node/basic_page/field_ref_resources/f7fde41d-ffca-4b2a-9b25-94b8a701a037/b33c8db5/20220706_resources_data_betten_03.pdf)

### OSI Status

I don't know

### License author or steward

The Cabinet Secretariat

### URL to project(s) that use license

https://www.digital.go.jp/copyright-policy/

**Comment by jlovejoy** [comment]
hi @sorairolake - looking at the terms again, this really seems to be terms of use for content on a website, which is a bit of a different request than we usually see for the SPDX License List. Most of the licenses cover software, hardware, data, and then there is the Creative Commons licenses which cover other copyrightable literary works. So, I'm just trying to really understand the context of the license. 

This license also seems to have a template-like structure. For example, sections 2b, 2c, and 3a allow for varying additional notices, etc. which could alter the license. 

Is there plans by the Government of Japan to use SPDX in some way, and that is the reason behind the submission?

**Comment by sorairolake** [comment]
@jlovejoy Yes, this license have a template-like structure. Emphasis in the license is replaced by each ministry with appropriate information.

I don't know if the government has such plans. The reason I submitted this is that SPDX includes licenses for free content such as the CC license. If this license is included in SPDX, I think it's useful when using content created by the government as part of software (such as images and data). This license is compatible with CC BY 4.0, so they could use the license identifier for CC BY 4.0, but I think it would be even better if this license is included in SPDX.

**Comment by jlovejoy** [comment]
# New submission review

* **Has the license been approved by the OSI?** _All OSI-approved license will be included on the SPDX License List, regardless of other factors_
    - [ ] Yes
    - [ ] No

## Definitive Factors

_These must all be satisfied to allow inclusion in the license list_

1. **Is the submitted license unique, that is, it does not match another license already on the License List as per the [matching guidelines](https://spdx.org/spdx-license-list/matching-guidelines)?**
    - [ ] Yes
    - [ ] No
1. **If a software license, does it apply to source code and not only to executables?**
    - [ ] Yes
    - [ ] No
1. **Does the license have identifiable and stable text, and is not in the midst of drafting?**
    - [ ] Yes
    - [ ] No
1. **Has the license steward, if any, committed to versioning new versions and to not modify it after addition to the list?**
    - [ ] Yes
    - [ ] No

## Other factors for inclusion

*Roughly in order of descending importance*

1. **Does the license substantially comply with one of the free/open content definitions? (examples include the [Open Source Definition](https://opensource.org/osd) and the [Debian Free Software Guidelines](https://www.debian.org/social_contract#guidelines))** (_Approval by the organisation that publishes the definition is not required_)
    - [ ] Yes
    - [ ] No
1. **Is the license structured to be generally usable by anyone, and not specific to one organisation or project?**
    - [ ] Yes
    - [ ] No
1. **Does the license have substantial use such that it is likely to be encountered (ie. use in many projects, or in one significant project)?** (_For recently written licenses, definitive plans for it to be used in at least one or a few significant projects may satisfy this_)
    - [ ] Yes
    - [ ] No
1. **Is the license primarily intended to facilitate the free distribution of content with limited restrictions?**
    - [ ] Yes
    - [ ] No
1. **Does the license steward support this submission, or is at least aware of and not in opposition of it?**
    - [ ] Yes
    - [ ] No

## Summary of factors, outcome, comments

_Comments on the totality of these "other factors" in light of the SPDX License List's overall goals and objectives_

**Comment by jlovejoy** [comment]
discussed on 6/22 call: Decided to not accept as the SPDX License List doesn't usually add website terms of use and there does not seem to be substantial use. If this ends up getting used more broadly or the gov't of Japan has interest in adding it to SPDX, then it could be re-submitted.

**Comment by sorairolake** [comment]
@jlovejoy Thanks! I understood the decision.
